<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrollbackSettingsPane</class>
 <widget class="QWidget" name="ScrollbackSettingsPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="noScrollbackRadioButton">
     <property name="text">
      <string>No Scrollback</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="limitScrollbackRadioButton">
     <property name="text">
      <string>Limit Scrollback to</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="scrollbackSpin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> lines</string>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="value">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="infiniteScrollbackRadioButton">
     <property name="text">
      <string>Infinite Scrollback</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
